BitFaster.Caching/Lru/AfterReadStopwatchPolicy.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace BitFaster.Caching.Lru
+{
+#if !NETCOREAPP3_0_OR_GREATER
+    /// <summary>
+    /// Implement an expire after read policy.
+    /// </summary>    
+    /// <remarks>
+    /// This class measures time using Stopwatch.GetTimestamp() with a resolution of ~1us.
+    /// </remarks>
+    public readonly struct AfterReadLongTicksPolicy<K, V> : IItemPolicy<K, V, LongTickCountLruItem<K, V>>
+    {
+        private readonly long timeToLive;
+
+        /// <summary>
+        /// Initializes a new instance of the TLruLongTicksPolicy class with the specified time to live.
+        /// </summary>
+        /// <param name="timeToLive">The time to live.</param>
+        public AfterReadLongTicksPolicy(TimeSpan timeToLive)
+        {
+            this.timeToLive = TLruLongTicksPolicy<K, V>.ToTicks(timeToLive);
```

Comment:
Time computation should be moved to an internal static class, rather than depending on the TLruPolicy.

Reply:
StopwatchTickConverter already exists.